bundles/EcommerceFrameworkBundle/CartManager/SessionCart.php

```diff
@@ -44,7 +44,7 @@ protected function getCartCheckoutDataClassName()
     protected static function getSessionBag(): AttributeBagInterface
     {
         /** @var AttributeBagInterface $sessionBag */
-        $sessionBag = \Pimcore::getContainer()->get('session')->getBag(SessionConfigurator::ATTRIBUTE_BAG_CART);
+        $sessionBag = \Pimcore::getContainer()->get('request_stack')->getSession()->getBag(SessionBagListener::ATTRIBUTE_BAG_CART);
```

Comment:
do we need a fallback there if there is no `request_stack` available? at least for BC reasons?

Reply:
done here https://github.com/pimcore/pimcore/pull/12241/files#diff-5800088a556ed69ab8bbc37c87de139807255039d98f4b6c1b18f01cdea221d3